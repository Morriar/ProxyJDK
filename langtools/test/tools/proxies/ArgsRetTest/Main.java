/*
 * @test
 * @summary tests args and return on primitive proxies
 * @author Alexandre Terrasa
 *
 * @compile RealSubject.java Subject.java Main.java
 */

package TMPProxyTests.ArgsRetTest;

public class Main {

	/**
	 * @param args
	 */
	public static void main(String[] args) {
		Subject rs = new RealSubject();
		Subject prxy = proxy Subject(rs);
		prxy.doHello();
		prxy.doHelloArgs("Alex", "Toto");
		System.out.println(prxy.doHelloRet());
		System.out.println(prxy.doHelloRetArgs("Alex", "Toto"));
	}
}
