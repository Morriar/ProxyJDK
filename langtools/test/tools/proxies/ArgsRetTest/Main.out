Hello
Hello Alex and Toto
Hello
Hello Alex and Toto